borg/helpers.py

```diff
@@ -1256,3 +1256,20 @@ def format_time(self, key, item):
 
     def time(self, key, item):
         return safe_timestamp(item.get(key) or item[b'mtime'])
+
+
+COMPRESS_NONE = set("""
+.gz .tgz .bz2 .tbz2 .xz .lzma .lzo .zip .rar .7z
+.mp3 .ogg .oga .flac
+.mp4 .mkv .avi .mpg .mpeg .webm .vob .ts .ogv
+.jpg .jpeg .png .gif
```

Comment:
I'd also like to exclude rpm files as they are already compressed (likely the same is true for .deb - maybe also .msi?)